SpinCore_pp/ppg/echo.py

```diff
@@ -96,13 +96,14 @@ def run_spin_echo(
     tx_phases = r_[0.0, 90.0, 180.0, 270.0]
     nPhaseSteps = len(ph1_cyc) * len(ph2_cyc)
     data_length = 2 * nPoints * nEchoes * nPhaseSteps
+    for_RX_nScans = 1
```

Comment:
nScans still needs to be it's own variable but this this prevents having to multiply by nScans in the post processing function.

Reply:
please `grep -Rile 'run_spin_echo'` in the examples directory to find the programs that use this function, and then just increment the version number of the post proc by 1 for all of those files. 

This way we know that they were acquired with a version that does not make us multiply by n scans in post-processing

Reply:
I incremented versions and ran into issues needing to do some cleanup for them to pass flake but should be good now